src/components/ErrorBoundary/ErrorBoundary.test.jsx

```diff
@@ -27,14 +29,13 @@ describe('<ErrorBoundary />', () => {
   });
   it('logs the error', () => {
     const welcomeError = new Error('Catch me if you can!');
-    NewRelicService.logError = jest.fn();
 
     const wrapper = shallow((
       <ErrorBoundary>
         <Welcome />
       </ErrorBoundary>
     ));
     wrapper.find(Welcome).simulateError(welcomeError);
-    expect(NewRelicService.logError).toHaveBeenCalledWith(new Error('Catch me if you can!'));
+    expect(logError).toHaveBeenCalledWith(new Error('Catch me if you can!'));
```

Comment:
this means the imported logger takes care of New Relic logging too? we have a ticket for abstracting out new relic, perhaps this pattern can be re used in that

Reply:
Correct, frontend-platform "owns" the New Relic logging through the `logInfo` and `logError` functions

Reply:
wohoo!